cli/send.go

```diff
@@ -120,6 +133,14 @@ var SendCmd = &cli.Command{
 			params.From = faddr
 		}
 
+		if params.From == address.Undef {
```

Comment:
```suggestion
		} else {
```

Perhaps there's a way that one of the above options can produce an `address.Undef`, if that's the case then we should probably error in those branches. Having this free-floating feels like it's going to catch a possible error case that should be reported to the user and we pick a default that may not be intended.